### ISSUE TITLE
Options single run proposition

### ISSUE THREAD

**Issue by ArturMarszalek** [fyodorvi/jspm-watch#12]
Hi,

I want to run build script only once to retrieve packaged files (app, tests). 
It will be nice to have option in buildOptions like 
`singeRun: true`
or add a simple method to the **watcher** like stop();
What do you think about this?

ps. great package, it helps me with TDD approach!

**Comment by fyodorvi** [comment]
@ArutrMarszalek not sure what you're trying to achieve. If you need packaged files you may use JSPM native bundling, that's exactly what this this module does, except that it watches for changes and invalidates changed files from cache.
